main.go

```
@@ -117,12 +117,14 @@ func main() {
 	if len(diffText) > 3000 {
 		diffText = diffText[:3000] + "\n...[truncated]"
 	}
+	fmt.Printf("diff: %v\n", diffText[:50])
 
 	// 调用OpenAI ChatGPT接口进行代码审查
 	review, err := callOpenAI(aiKey, diffText)
```

Silently converting API failures to "failed" string masks errors and reduces reliability of the review system.